website/docs/cli/code/index.mdx

```diff
@@ -2,7 +2,7 @@
 page_title: Writing and Modifying Code - Terraform CLI
 description: >-
   Learn commands that help validate, format, and upgrade code written in the
```

Comment:
```suggestion
  Learn commands that help validate, format, and upgrade code written in 
```

oups, I missed this. 

"Terraform's configuration language" is also used here: https://developer.hashicorp.com/terraform/language/syntax/configuration